src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -138,6 +145,8 @@ public AddressBook load() throws StorageOperationException {
             throw new StorageOperationException("Error parsing file data format");
         } catch (IllegalValueException ive) {
             throw new StorageOperationException("File contains illegal data values; data type constraints not met");
+        } catch (RuntimeException rte) {
+        	throw new FileDeletedWhileRunningException("File deleted while program is running");
```

Comment:
If the file is missing when we tried to load (for example, when we first run the program) and we can recover from that, can we recover from this Exception as well?